Sources/SwiftKuery/Table.swift

```diff
@@ -159,19 +294,53 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
-    /// - Returns: A new instance of `Table`.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, the primary key is set to the `id` column for the table `personTable`.
+     ```swift
+     public class PersonTable: Table {
+        let tableName = "personTable"
+        let id = Column("id", Int32.self, notNull: true)
+        let firstName = Column("firstName", String.self, notNull: true)
+        let lastName = Column("lastName", String.self, notNull: true)
+     }
+     var personTable = PersonTable()
+     personTable = personTable.primaryKey(idColumn)
```

Comment:
Doesn't this need to be `personTable = personTable.primaryKey(personTable.id)` now that you've removed idColumn from the example?